hack/release/pkg/builder/builder.go

```diff
@@ -257,6 +261,26 @@ func (r *ReleaseBuilder) collectGithubArtifacts(ver string) error {
 		return err
 	}
 
+	// Generate a SHA256SUM file containing the checksums for each artifact
+	// that we attach to the release. These can be confirmed by end users via the following command:
+	// sha256 -c --ignore-missing SHA256SUMS
+	files, err = ioutil.ReadDir(uploadDir)
+	if err != nil {
+		return err
+	}
+	sha256args := []string{}
+	for _, f := range files {
+		sha256args = append(sha256args, f.Name())
+	}
+	output, err := r.runner.RunInDir(uploadDir, "sha256sum", sha256args, nil)
```

Comment:
I didn't know this before but sha256sum will fail if given directories (which it won't be since these are all artifacts to upload).